### ISSUE TITLE
Save gets corrupted on switching apps mid-turn

### ISSUE THREAD

**Issue by Similacrest** [hmdzl001/SPS-PD#3]
I have experienced multiple crashes with save corruption while switching apps while the turn is ongoing. Also suspect it might be pet-related, as I've had at least one occurence of a save corruption just from me and pet attacking, but with no pets in play the game was stable.

SPS-PD 0.9.4 Year hunter
Android 10 / MIUI11 20.1.21 / rooted (battery saver is off for SPS-PD, game booster on).

Let me know if I can provide any additional info, for instance I can export the corrupted saves with Titanium Backup?..

**Comment by hmdzl001** [comment]
Yes... year hunter verison is easy to crash, and I fix it in next verison.
